src/style.scss

```diff
@@ -69,6 +69,10 @@ p {
   min-height: 100vh;
 }
 
+#content {
+  min-height: calc(100vh - 103px)
```

Comment:
How do you get 103px? Can't you just use the class "full-height" from Primer?

Reply:
103px is the height of the navbar, without removing it from the total height we will get horizontal scroll bar 😢